packages/components/src/components/carousel/__snapshots__/carousel-styles.spec.ts.snap

```diff
@@ -340,21 +340,32 @@ h2,::slotted([slot=heading]),p,::slotted([slot=description]) {
   -webkit-user-select: none;
   -webkit-touch-callout: none;
 }
-.pagination {
+.pagination-container {
   display: flex;
+  position: relative;
   justify-content: center;
+  width: calc(20px + 8px * 4 + 8px * 4);
+  left: calc(50% - 42px);
+  overflow-x: hidden;
+}
+.pagination {
+  display: flex;
+  align-items: center;
+  width: fit-content;
   gap: 8px;
+  transition: all 400ms;
+}
+.pagination > span.bullet--active {
```

Comment:
Let's try to get rid of this unnecessary chaining. `.bullet-active` should be all we need for a selector (maybe with `!important` keyword) because of specificity.